dev/src/components/teacher/EditBenchmark.tsx

```diff
@@ -1,35 +1,40 @@
-import { json } from "overmind"
 import React, { useState } from "react"
 import { Assignment, GradingBenchmark } from "../../../proto/ag/ag_pb"
+import { Converter } from "../../convert"
 import { useActions } from "../../overmind"
 
 
-const EditBenchmark = ({ children, benchmark, assignment }: { children?: React.ReactNode, benchmark?: GradingBenchmark, assignment: Assignment }): JSX.Element => {
+const EditBenchmark = ({ children, benchmark, assignment }: { children?: React.ReactNode, benchmark?: GradingBenchmark.AsObject, assignment: Assignment.AsObject }): JSX.Element => {
     const actions = useActions()
 
     const [editing, setEditing] = useState<boolean>(false)
     const [add, setAdd] = useState<boolean>(benchmark ? false : true)
-    const [newBenchmark, setNewBenchmark] = useState<GradingBenchmark>(new GradingBenchmark().setAssignmentid(assignment.getId()))
-    const bm = json(benchmark) ?? newBenchmark
-    const copy = json(benchmark)?.cloneMessage()
+
+    // Clone the criterion, or create a new one if none was passed in
+    const bm = benchmark
+        ? Converter.clone(benchmark)
```

Comment:
There were a few other uses of the ternary operator: `? :` where the `?` was on the line above, while the `:` was like this example. I prefer this style where the `?` and `:` line up on top of each other.